### DISCUSSION TITLE
Adding Create Subasset to the Right Click Token Dropdown

### DISCUSSION THREAD

**Discussion by davestaxcp** [jdogresorg/freewallet-desktop#176] [Ideas]
If a user is looking to mint a subasset for a specific token they have token ownership for the process feel a bit to "manual" instead of an easy automatic process that the program is showing you how a subasset works:

![image](https://github.com/jdogresorg/freewallet-desktop/assets/136373423/1682cf69-0301-4c96-923a-e960f0c7fde7)

Can we add "Create Token Subasset of TOKENNAME" to this list? ^

And when it shows this page for token subasset:

![image](https://github.com/jdogresorg/freewallet-desktop/assets/136373423/fea55e16-60fd-404d-bf56-02b85d3aa382)

Subasset is automatically picked as well as this part is AUTO filled so it forces the user to do BIGGIEPEPE dot [insert subasset name here]

Basically just this would be "locked" so the user is just focusing on the subasset name:

![image](https://github.com/jdogresorg/freewallet-desktop/assets/136373423/751b26f7-d0dc-434f-bc3a-a59d804f0f5d)

It would be much less confusing to a new user. 

Just my two sats <3

**Comment by jdogresorg** [comment]
added the `Create XXX Subasset` option with commit https://github.com/jdogresorg/freewallet-desktop/commit/52043331e8041768d27d42b36b18e93db62aabd9

Should be in the next release.
